hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/NameNode.java

```diff
@@ -2663,6 +2663,8 @@ String reconfigureSlowNodesParameters(final DatanodeManager datanodeManager,
             DFS_DATANODE_PEER_STATS_ENABLED_DEFAULT :
             Boolean.parseBoolean(newVal);
         result = Boolean.toString(peerStatsEnabled);
+        getConf().setBoolean(DFS_DATANODE_PEER_STATS_ENABLED_KEY,
+            peerStatsEnabled);
```

Comment:
remove this, we explore this in separate  for all configs, if needed

Reply:
I have removed this.